sunpy/map/sources/mlso.py

```diff
@@ -0,0 +1,64 @@
+import matplotlib.pyplot as plt
+
+from astropy.visualization.mpl_normalize import ImageNormalize
+from astropy.visualization import PowerStretch
+import astropy.units as u
+
+from sunpy.map import GenericMap
+from sunpy.map.sources.source_type import source_stretch
+from sunpy.coordinates import get_sunearth_distance
+
+__all__ = ['KCorMap']
+
+
+class KCorMap(GenericMap):
+    """
+    K-Cor Image Map.
+
+    The COronal Solar Magnetism Observatory (COSMO) K-coronagraph (K-Cor) is one of three proposed
+    instruments in the COSMO facility suite. It is specifically designed to study the formation
+    and dynamics of coronal mass ejections and the evolution of the density structure of
+    the low corona. The K-Cor records the polarization brightness (pB) formed by Thomson scattering
+    of photospheric light by coronal free electrons. The National Center for Atmospheric
+    Research (NCAR), via the National Science Foundation (NSF), provided full funding for the
+    COSMO K-Cor, which was deployed to the Mauna Loa Solar Observatory (MLSO) in Hawaii in
+    September 2013, replacing the aging MLSO Mk4 K-coronameter.
+
+    References
+    ----------
+    * `COSMO Mission Page <https://www2.hao.ucar.edu/cosmo>`_
+    * `KCOR Instrument Page <https://www2.hao.ucar.edu/mlso/instruments/mlso-kcor-coronagraph>`_
+    """
+
+    def __init__(self, data, header, **kwargs):
+
+        GenericMap.__init__(self, data, header, **kwargs)
```

Comment:
This should be like this now:

```suggestion
        super()(data, header, **kwargs)
```